doc/faq.rst

```diff
@@ -180,11 +180,8 @@ Do you support PyPy?
 --------------------
 
 In case you didn't know, `PyPy <http://pypy.org/>`_ is the new, fast,
-just-in-time compiling Python implementation. We don't support it.
-When the `NumPy support <http://buildbot.pypy.org/numpy-status/latest.html>`_
-in PyPy is complete or near-complete, and SciPy is ported over as well,
-we can start thinking of a port.
-We use too much of NumPy to work with a partial implementation.
+just-in-time compiling Python implementation. Experimental support for
```

Comment:
I would rather say: "`PyPy <http://pypy.org/>`_ is the new, fast,  is an alternative Python implementation with a built-in just-in-time compiler."

It's not that new anymore. It is fast in many case but for scikit-learn users it's quite unlikely to improve speed significantly on our models as they already rely on numpy and Cython.